app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -263,6 +263,12 @@ def parse_date(year:, month:, day:)
           if year.to_i.positive? && month.to_i.positive? && day.to_i.positive?
             Date.new(year.to_i, month.to_i, day.to_i).to_s
           end
+        rescue
```

Comment:
I'd consider rescuing `ArgumentError` specifically, but it may not matter
```suggestion
        rescue ArgumentError
```

Reply:
Gotcha - done in 697d7b79b.